src/generators/static/rollupConfigFileGenerator.ts

```diff
@@ -5,12 +5,17 @@ import { Project } from "ts-morph";
 import { getAutorestOptions } from "../../autorestSession";
 
 export function generateRollupConfig(project: Project) {
-  const { generateMetadata } = getAutorestOptions();
+  const { generateMetadata, restLevelClient, azureSdkForJs } = getAutorestOptions();
 
   if (!generateMetadata) {
     return;
   }
 
+  // when it's generating rlc codes, only generate rollup config in codegen test
```

Comment:
does `dev-tool` includes rollup now? If yes, I think it's better to keep rollup because we should try our best to make the generated codes for codegen test have the same build/pack environment as the ones in `azure-sdk-for-js`.

Reply:
IMO, if we are adding an `azure-sdk-for-js` that's where we'd try to align with the repo. However, for stand-alone generated packages, I think we should try to generate packages as tooling agnostic as possible so that any external users can choose what build/bundling/etc tooling they want to use.

If we are not expecting users running autorest without `azure-sdk-for-js` flag to use the generated code in the mono repo, what's the point of trying to align with the tools using there?

@sarangan12 do you have any thoughts on this?

Reply:
In the `azure-sdk-for-js` repository, the standard is to use rollup. So, if we design an option (such as `azure-sdk-for-js`) then it is logical to design the option to the same standard. So, when this option is used, the generator should be using the `dev-tool` and NOT the regular rollup.

On the other hand, if that option is not used, then the assumption is that the SDK is generated as a stand alone project. In such a case, we should be using the rollup.

Reply:
So I think rollup config file is needed. right?

Reply:
For stand-alone, we need it. For the ones with the flag `azure-sdk-for-js` set, we do not need it.

Reply:
I think the current implementation is generating rollup files when set flag `--azure-sdk-for-js=false`, and not generating it when generating codes foor azure-sdk-for-js. It is same as @sarangan12 's idea. thanks